docs/index.rst

```diff
@@ -0,0 +1,33 @@
+AutoGluon-Cloud: Train and Deploy AutoGluon on the Cloud
+================================================================
+
+AutoGluon-Cloud aims to provide user tools to train, fine-tune and deploy [AutoGluon](https://auto.gluon.ai/stable/index.html) backed models on the cloud. With just a few lines of codes, users could train a model and perform inference on the cloud without worrying about MLOps details such as resource management.
+
+Currently, AutoGluon-Cloud supports [AWS SageMaker](https://aws.amazon.com/sagemaker/) as the cloud backend.
+
+.. note::
+
+    Example using AutoGluon-Cloud to train and deploy AutoGluon backed model on AWS SageMaker:
+
+    >>> predictor_init_args = {label='label'}  # init args you would pass to AG TabularPredictor
+    >>> predictor_fit_args = {train_data, time_limit=120}  # fit args you would pass to AG TabularPredictor
```

Comment:
For the completeness of the example, can we add codes for constructing the `train_data` and the `test_data` from public repo? Users should be able to copy paste these example codes and get things running with minimal modifications

Reply:
Sounds good